### PR TITLE
improved page loading time

### PR DESCRIPTION
**Pull request by ttraenkler**
addresses the low hanging fruits of #52 (page loading time).

- removed inlined background heightlines svg from _app.js
- made heightlines.svg a separate file
- added webpack bundle analyzer

### CODE REVIEW

**Comment by ttraenkler** [comment]
the Minimize main-thread work 2.1 s warning disappeared in the lighthouse report

lighthouse score:
Bildschirmfoto 2020-01-15 um 01 19 04

before:
Bildschirmfoto 2020-01-15 um 00 54 36

after:
Bildschirmfoto 2020-01-15 um 00 54 43

**Comment by ttraenkler** [comment]
Updated the screenshots to make the two conditions more comparable, instead of comparing the live site with a production build of localhost i compared dev and the branch on localhost under the same conditions to exclude a localhost advantage

File: pages/_app.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const GlobalStyles = createGlobalStyle`
...
...-13.06 14.17-21.69 1.58-1.32 6.67-5.44 7.09-5.78a48.03 48.03 0 0 0 5.23-4.77c4.1-4.63 5.85-9.55 7.8-20.07a501.52 501.52 0 0 0 .8-4.37c.33-1.87.6-3.3.88-4.73.74-3.78 1.5-7.18 2.4-10.63 1-3.78 1.38-5.5 2.36-10.37.6-3.02.93-4.21 1.56-5.47 1.22-2.45 1.27-2.5 12.25-11.42zm.64.78c-10.77 8.74-10.88 8.84-12 11.08-.58 1.16-.88 2.3-1.47 5.22-.98 4.89-1.36 6.63-2.37 10.44-.9 3.43-1.65 6.8-2.39 10.56a339.79 339.79 0 0 0-1.29 6.95l-.39 2.15c-1.98 10.68-3.77 15.74-8.04 20.54a48.77 48.77 0 0 1-5.34 4.88c-.42.34-5.5 4.47-7.07 5.78-10.04 8.4-14.72 14.65-13.83 20.78.45 3.1 2.44 6.03 6.17 8.83 3 2.25 5.39 4.62 7.24 7.12 4.63 6.24 5.52 11.52 5.15 21.15-.25 6.14-.01 8.1 1.66 9.78 6.1 6.1 31.02 1.33 42.31-9.02 2.75-2.52 6.66-5.43 13.16-9.92l8.6-5.89c3.63-2.48 6.45-4.44 9.19-6.4 14.73-10.54 24.44-19.18 29.97-27.7 4.9-7.54 15.31-26.68 16.02-27.8 3.27-5.26 6.26-8.41 10.18-10.37 6.79-3.4 16.65-9.63 25.03-16.32 11.52-9.18 19.61-18.53 22.72-27.4 3.07-8.78-2.02-20.27-12.52-31.49-8.8-9.4-20.04-16.84-25.01-16.84-10.67 0-25.43 5.5-35.68 13.89-3.76 3.07-7.9 4.81-13.5 6.09-1.41.32-2.35.5-5.11 1.02-8.21 1.55-11.76 2.73-15.38 5.88-3.34 2.9-6.45 4.22-12.7 5.92-11.26 3.07-15.75 4.94-23.31 11.09zM212 251.85c0 7.56-.6 10
```
Comment (torch2424) [inline pages/_app.tsx]: Thank you for moving this out 😄

**Comment by ttraenkler** [comment]
Thanks for the merge. I verified the lighthouse score (mobile slow 4g 4x slowdown) after deployment:

Bildschirmfoto 2020-01-15 um 14 16 13

**Pull request merged**